packages/stable/src/api/geospatial/types.ts

```diff
@@ -269,3 +269,38 @@ export interface GeospatialCRSResponse
 export interface GeospatialSridId {
   srid: number;
 }
+
+export type GeospatialComputeFunction =
+  | {
+      stTransform: GeospatialGeometryTransformComputeFunction;
+    };
+
+export interface GeospatialGeometryTransformComputeFunction {
+  geospatialGeometryComputeFunction: GeospatialGeometryComputeFunction;
+  srid: number;
+}
+
+export type GeospatialGeometryComputeFunction =
+  | {
+      geometryValue: GeospatialGeometryValueComputeFunction;
```

Comment:
suggestion(blocking): the key here should be `ewkt`

Reply:
That has been replaced with an interface